### DISCUSSION TITLE
Web3Modal Error

### DISCUSSION THREAD

**Discussion by gtsonevv** [WalletConnect/walletconnect-monorepo#1941] [Developer Support]
For some reason I get this error
```
Error: ClientCtrl has no client set
```
I'm using the code from [here](https://github.com/WalletConnect/web3modal/blob/V2/examples/react/src/pages/_app.tsx). If I remove `Web3Modal` component the error is gone. Any idea what might be causing that?

**Comment by chrisbergs** [comment]
I have the same problem...

**Reply by xbeniao** [reply]
Has the problem been resolved？

**Comment by canab** [comment]
The same issue

**Reply by xbeniao** [reply]
Has the problem been resolved？

**Comment by 70nyIT** [comment]
I've just created a more details post as I'm having the same issue. Happy I'm not the only one. Are you guys able to reproduce the issue in CodeSandbox? See my post about similar issue here: https://github.com/WalletConnect/walletconnect-monorepo/discussions/2083
